### DISCUSSION TITLE
displaying rtsp video stream in a web page

### DISCUSSION THREAD

**Discussion by Farzinkh** [bluenviron/mediamtx#505] [Questions] [answered]
I was just wondering if you can recommend something to me or guide me that how i can display rtsp stream video on web page i just want to display live cameras on browser in the simplest form.

**Comment by benbullnz** [comment, marked as answer]
This project is probably exactly what you are after. 

https://github.com/deepch/RTSPtoWeb

**Comment by aler9** [comment]
Since there's now a HLS server integrated into rtsp-simple-server, you can add the stream to the configuration file:

```yml
paths:
  stream:
    source: rtsp://sourceurl
```

And then visit

```
http://localhost:8888/stream
```

**Comment by aler9** [comment]
BTW, the rtsp-simple-server approach also supports audio, which `RTSPtoWeb` doesn't.

**Reply by ikcehcus** [reply]
are you sure that rtsp simple server supports audio? 
when I do:

`
paths:
  stream:
    source: rtsp://sourceurl
`
    
 in vlc rtsp://sourceurl contains video and audio

then in  browser

`http://localhost:8888/stream`

contains only video

**Reply by aler9** [reply]
Audio is fully supported but only when audio is encoded with AAC, since HLS supports only AAC.
If your stream uses a different encoding, you can convert it to AAC by following instructions in the readme:

https://github.com/aler9/rtsp-simple-server#remuxing-re-encoding-compression

[a comment by an automated account is not shown]
